openpathsampling/engines/dynamics_engine.py

```diff
@@ -262,8 +268,8 @@ def __getattr__(self, item):
                     # alternately, trust the fixed result with
                     # return result  # miraculously fixed
                     raise AttributeError(
-                        "Unknown problem occurred in property" + 
-                        str(p.fget.func_name) + ": Second attempt returned"
+                        "Unknown problem occurred in property"
+                        + property_name(p) + ": Second attempt returned"
```

Comment:
thanks, will do. Yes, I tried that in both and it seemed to work... No idea why I did not use that in the first place...